dataset-variance-resub-2022-edges/main/feature_defs.json

```diff
@@ -220,7 +228,7 @@
     "description": "ID of shape-matched cell (available for edge cells only).",
     "tooltip": "ID of shape-matched cell (available for edge cells only).",
     "unit": "unitless",
-    "key": "type",
+    "key": "neigh",
```

Comment:
`neigh` should be the key for `"displayName": "Neighbor ID"`, not `Cell class`. Although it seems to be correct in the viewer. Have you committed your last version?

Reply:
it's correct here. the github interface is making it look like the change was on "cell class" but it really isn't.  There's a spacer in between that's hiding some lines in the github interface